### ISSUE TITLE
Ubuntu Asusctl dont work

### ISSUE THREAD

**Issue by P-C-R** [drunckj/Asus-ESS#1]
The install of asusctl dont work. Asusctl has no ubuntu support anymore. You have to build it yourself ;(

**Comment by drunckj** [comment]
yeah they stopped maintaining packages for ubuntu due to packaging issues. I will change the ubuntu option to build it from source.

**Comment by SalahAdDin** [comment]
> yeah they stopped maintaining packages for ubuntu due to packaging issues. I will change the ubuntu option to build it from the source.

Is it the same as installing it by ourselves?

**Comment by drunckj** [comment]
Yes but I'll automate the process and after build cleanup so you don't have to do each step one by one.

> > yeah they stopped maintaining packages for ubuntu due to packaging issues. I will change the ubuntu option to build it from source.
> > is it the same than installing it by ourselves?
